speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/BrakeDecision.cpp

```diff
@@ -1,11 +1,13 @@
 #include "BrakeDecision.h"
-
-void BrakeDecision::GetIndicateCommands()
+#include "Mediator.h"
+void BrakeDecision::RunIndicateCommands()
 {
+	if (m_brakeAmount < 0.5f) return;
```

Comment:
Where does the value 0.5 come from

Reply:
it just needed a value and can be changed later